### PR TITLE
docs: fix typo in "deploy multiple locales" section

### PR DESCRIPTION
**Pull request by gobika21**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #42540

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/guide/i18n.md [hunk 1]
```diff
@@ -748,7 +748,7 @@ Specify the warning level in the `options` section for the `build` target of you
 
 ## Deploy multiple locales
 
-If `myapp` is the directory containing your app's distributable files, you would typically make available different versions for different locales in locale directories such as `myapp/fr` for the French version and `myapp.com/es` for the Spanish version.
+If `myapp` is the directory containing your app's distributable files, you would typically make available different versions for different locales in locale directories such as `myapp.com/fr` for the French version and `myapp.com/es` for the Spanish version.
```
Comment (dario-piotrowicz) [inline aio/content/guide/i18n.md:751]: The `.com` of the _es_ version be removed too, shouldn't it?

**Comment by dario-piotrowicz** [comment]
I think that you're not following the commit message convention (https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-header)

File: aio/content/guide/i18n.md (hunk 1 above)
Comment (petebacondarwin) [inline aio/content/guide/i18n.md:751]: This paragraph is talking about the directories where the files are deployed not URLs, so actually the `.com` should be removed in both cases.

```suggestion
If `myapp` is the directory containing your app's distributable files, you would typically make available different versions for different locales in locale directories such as `myapp/fr` for the French version and `myapp/es` for the Spanish version.
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks for making this PR @gobika21. I think the opposite change is actually needed.
Also, when fix this, you amend the commit message to follow our conventions - something like:

```
docs: fix typo in "deploy multiple locales" section

Fixes #42540
```

**Comment by dario-piotrowicz** [comment]
@gobika21 please do not create extra commits but rebase just into one with the correct commit message

If you don't know how to do it I can provide assistance

File: aio/content/guide/i18n.md (hunk 1 above)
Comment (dario-piotrowicz) [reply aio/content/guide/i18n.md]: ah yeah sorry I commented the wrong way around here, I meant to remove it sorry! :P

Comment (dario-piotrowicz) [reply aio/content/guide/i18n.md]: yeah sorry I just wrote my comment the other way around by mistake, sorry about that :P

**Comment by gobika21** [comment]
Thanks for the review @petebacondarwin and @dario-piotrowicz 
I have addressed the review comments. Please check now.

**Review by petebacondarwin — APPROVED** [review]
LGTM! Thanks

Reviewed-for: global-docs-approvers

**Comment by dario-piotrowicz** [comment]
Thanks @gobika21 looks all good now :smiley:

**Pull request closed without merging**